### PR TITLE
refactor/mobxAutorun/todos

### PR DESCRIPTION
**Pull request by SerkanSipahi**
- clarification refactor for *mobxAutorun directive(s) in todos template
- update package.json

### CODE REVIEW

**Comment by SerkanSipahi** [comment]
Build is [failed](https://travis-ci.org/mobxjs/mobx-angular/builds/336903579?utm_source=github_status&utm_medium=notification) due to something in the root package.json of mobx-angular! it has nothing to do with my pull-request!

File: example/todos/src/app/components/count/count.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Todos } from '../../stores/todos.store';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'app-count',
+  template: `
+    <span id="todo-count" *mobxAutorun><strong>{{todos.uncompletedItems}}</strong>
```
Comment (SvenSchoene) [inline example/todos/src/app/components/count/count.component.ts]: Personally, I'd put the `<strong>...</strong>` on its own line.

Comment (SerkanSipahi) [reply example/todos/src/app/components/count/count.component.ts]: done!

**Comment by adamkleingit** [comment]
Thanks! merged

**Pull request merged**